tests/configs/standard/output/lab-objects.json

```diff
@@ -1,575 +1,586 @@
 [
-  {
```

Comment:
It looks like this whole file changed indentation from two spaces to four spaces. That makes it unreviewable, since GitHub has no idea what changed, and that in turn means that I can't be confident we didn't break the existing lab spawning because we don't have consistent test data. Could you put this back to the indentation it had before, or if you need to change the indentation for some reason, do that as a separate PR? Since I think we should merge most of this code without detailed review, I want to be sure that the existing lab spawning test suite was run and we're really sure we didn't break anything in it.

Reply:
Oh, huh.  Yeah, I can change that to indent=2.  That's probably an editor or a dump setting that I got wrong.

Reply:
Done